Speak-like-Yoda/requirements.txt

```diff
@@ -0,0 +1,10 @@
+# Speak-Like-Yoda
```

Comment:
why requirements file contain readme text?

Reply:
@kaustubhgupta by mistake I wrote on the requirement.txt